src/Raven.Server/Web/System/BackupServerWideHandler.cs

```diff
@@ -0,0 +1,230 @@
+                await ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, newIndex);
+               
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                using (context.OpenReadTransaction())
+                {
+                    var result = new ModifyServerWideBackupResult()
+                    {
+                        TaskName = name,
+                        RaftCommandIndex = newIndex 
+                    };
+
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.OK;
+                    context.Write(writer, result.ToJson());
+                    writer.Flush();
+                }
+            }
+        }
+
+        // Get all server-wide backups -OR- specific task by the task name... 
+        // todo : consider also returning each db specific details for the studio list view here
+        [RavenAction("/admin/configuration/server-wide/backup", "GET", AuthorizationStatus.ClusterAdmin)]
+        public Task GetServerWideBackupConfigurationCommand()
+        {
+            var taskName = GetStringQueryString("name", required: false);
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var backups = ServerStore.Cluster.GetServerWideBackupConfigurations(context, taskName);
+                ServerWideBackupConfigurationResults backupsResult = new ServerWideBackupConfigurationResults();
+                
+                foreach (var backupBlittable in backups)
+                {
+                    var backup = JsonDeserializationServer.ServerWideBackupConfiguration(backupBlittable);
+                    backup.BackupDestinations = backup.GetDestinations();
+                    backupsResult.Results.Add(backup);
+                }
+                
+                context.Write(writer, backupsResult.ToJson());
+                writer.Flush();
+
+                return Task.CompletedTask;
+            }
+        }
+        
+        // Toggle Disable/Enable for specified task
+        [RavenAction("/admin/configuration/serve-wide/backup/state", "POST", AuthorizationStatus.ClusterAdmin)]
+        public async Task ToggleTaskState()
+        {
+            var disable = GetBoolValueQueryString("disable") ?? true;
+            var taskName = GetStringQueryString("taskName", required: false);
+           
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                // Get existing task
+                var backups = ServerStore.Cluster.GetServerWideBackupConfigurations(context, taskName); 
```

Comment:
Can you have more than one?

Reply:
If yes, then we need to switch all of them, but it should be unique IIRC?

Reply:
1. Yes - there is just 'one' - I will refactor this.
2. But, actually, we need to add server code that prevents creating a new task with an existing task name - as it is currently Not handled and new task with an existing name overrides the existing task